src/components/route/addBook.jsx

```diff
@@ -1,36 +1,37 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
-import '../css/bookStyle.css';
+import { useDispatch } from 'react-redux';
+import { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBookToApi, addBook } from '../../redux/books/booksSlice';
 
-const BookForm = ({ addBook }) => {
+const AddBook = () => {
+  const dispatch = useDispatch();
   const [title, setTitle] = useState('');
   const [author, setAuthor] = useState('');
-  const handleSubmit = (e) => {
-    e.preventDefault();
-    if (title && author) {
-      addBook({
-        title,
-        author,
-      });
-      setTitle('');
-      setAuthor('');
-    }
+
+  const data = {
+    item_id: uuidv4(),
+    title,
+    author,
+    category: 'Science-fiction',
+  };
+  const handleSubmit = () => {
+    dispatch(addBookToApi(data));
+    dispatch(addBook(data));
```

Comment:
:red_circle: [REQUIRED] I noticed that after clicking the add book button. The book  is rendered twice when a new book is added, Please 
![Peek 2023-09-15 14-43](https://github.com/misikir21/bookstore/assets/26736582/095eb404-a619-43f0-a171-19a134f65d8b)
ensure that only one book is rendered per addition. :arrow_down: